### ISSUE TITLE
From v-c.o

### ISSUE THREAD

**Issue by bsdphk** [varnishcache/varnish-cache#2312]
```
Assert error in vsl_sanity(), cache/cache_shmlog.c line 61:
  Condition((vsl->wlp) != 0) not true.
version = varnish-5.1.1 revision 308843aaf, vrt api = 6.0
ident = FreeBSD,11.0-RELEASE-p8,amd64,-jnone,-smalloc,-smalloc,-hcritbit,kqueue
now = 536695.958740 (mono), 1492024128.481095 (real)
Backtrace:
  0x453831: <pan_backtrace+0x31> at /usr/local/sbin/varnishd
  0x453637: <pan_ic+0x2d7> at /usr/local/sbin/varnishd
  0x464b2f: <vsl_sanity+0x9f> at /usr/local/sbin/varnishd
  0x465a77: <VSLb_ts+0x27> at /usr/local/sbin/varnishd
  0x45f090: <VSLb_ts_req+0xa0> at /usr/local/sbin/varnishd
  0x45ef33: <cnt_transmit+0x593> at /usr/local/sbin/varnishd
  0x45a1ef: <CNT_Request+0x73f> at /usr/local/sbin/varnishd
  0x49d3ce: <h2_do_req+0x25e> at /usr/local/sbin/varnishd
  0x483574: <Pool_Work_Thread+0x604> at /usr/local/sbin/varnishd
errno = 32 (Broken pipe)
thread = (cache-worker)
thr.req = 0x80b5a71a0 {
  MAGIC 0x00000000 (Should:REQ_MAGIC/0x2751aaa1)
  vxid = 0, transport = NULL
  step = 0x0,
  req_body = R_BODY_INIT,
  restarts = 0, esi_level = 0,
  ws = 0x80b5a7388 {
    MAGIC 0x00000000 (Should:WS_MAGIC/0x35fac554)
    id = \"\",
    {s, f, r, e} = {0x0, +0, +0, +0},
  },
  http[req] = 0x0 {
  },

```

**Comment by daghf** [comment]
Possibly related to or duplicate of #2310 - I saw this assert as well while working on that one.

**Comment by daghf** [comment]
Closing this as a duplicate of #2310.

Fixed in 5e98181a6180a50df0836539b2bc12b2264fb8fe
